virtual_ecosystem/models/animal/animal_territories.py

```diff
@@ -0,0 +1,248 @@
+"""The ''animal'' module provides animal module functionality."""  # noqa: #D205, D415
+
+from virtual_ecosystem.models.animal.animal_cohorts import AnimalCohort
+from virtual_ecosystem.models.animal.decay import CarcassPool, ExcrementPool
+from virtual_ecosystem.models.animal.plant_resources import PlantResources
+
+
+class AnimalTerritory:
```

Comment:
So, as a sort of summary, all the iteration between an animal cohort and its surroundings to thrive - or die - now happens here, right?